bin/cyhy-import

```diff
@@ -119,9 +119,14 @@ def import_request(db, request, source, force=False, init_stage=None):
 
 
 def import_file(db, filename, force, init_stage=None):
-    f = open(filename, "rb")
-    request = json.load(f)
-    f.close()
+    try:
+        with open(filename, "r") as f:
+            # For py3, encoding should move from json.load into open statement
+            request = json.load(f, encoding="ascii")
+    except UnicodeDecodeError as e:
+        print ("Document contained a non-ASCII character: {}".format(e))
```

Comment:
I would change "contained" to "contains", since the other output messages in this code appear to use the present tense.  See, for example, line 112 above.

Reply:
Done! Thanks for the suggestion.